lib/Parser/Turtle.php

```diff
@@ -1214,38 +1203,16 @@ protected function peek()
 
     /**
      * Steps back, restoring the previous character read() to the input buffer
-     */
-    protected function unread($chars)
-    {
-        $this->column -= mb_strlen($chars, "UTF-8");
-        $this->bytePos -= strlen($chars);
-        if ($this->bytePos < 0) {
-            $this->bytePos = 0;
-        }
-        if ($this->column < 1) {
-            $this->column = 1;
-        }
-    }
-
-    /**
-     * Reverse skips through whitespace in 4 byte increments.
-     * (Keeps the byte pointer accurate when unreading.)
      * @ignore
      */
-    protected function unskipWS()
+    protected function unread($c)
     {
-        if ($this->bytePos - 4 > 0) {
-            $slice = substr($this->data, $this->bytePos - 4, 4);
-            while ($slice != '') {
-                if (!self::isWhitespace(mb_substr($slice, -1, 1, "UTF-8"))) {
-                    return;
-                }
-                $slice = substr($slice, 0, -1);
-                $this->bytePos -= 1;
-            }
-            // This 4 byte slice was full of whitespace.
-            // We need to check that there isn't more in the next slice.
-            $this->unSkipWS();
+        # FIXME: deal with unreading new lines
```

Comment:
How important is this fix?

Reply:
@njh in my fork `git blame` claims this line was added by you five years before I made this change:
```
1c60f1f4 lib/EasyRdf/Parser/Turtle.php (Nicholas J Humfrey 2012-07-27 09:44:05 +0100 1204)     /**
1c60f1f4 lib/EasyRdf/Parser/Turtle.php (Nicholas J Humfrey 2012-07-27 09:44:05 +0100 1205)      * Steps back, restoring the previous character read() to the input buffer
1c60f1f4 lib/EasyRdf/Parser/Turtle.php (Nicholas J Humfrey 2012-07-27 09:44:05 +0100 1206)      * @ignore
1c60f1f4 lib/EasyRdf/Parser/Turtle.php (Nicholas J Humfrey 2012-07-27 09:44:05 +0100 1207)      */
1b2f54a4 lib/EasyRdf/Parser/Turtle.php (Nicholas J Humfrey 2013-08-04 19:24:04 +0100 1208)     protected function unread($c)
e714bc34 lib/EasyRdf/Parser/Turtle.php (Nicholas J Humfrey 2012-07-23 09:57:00 +0100 1209)     {
cce8ee01 lib/EasyRdf/Parser/Turtle.php (Nicholas J Humfrey 2013-08-20 19:39:33 +0100 1210)         # FIXME: deal with unreading new lines
545cb24f lib/Parser/Turtle.php         (Mateusz Żółtak     2018-07-19 16:25:56 +0200 1211)         $len = mb_strlen($c, "UTF-8");
545cb24f lib/Parser/Turtle.php         (Mateusz Żółtak     2018-07-19 16:25:56 +0200 1212)         $this->column -= $len;
545cb24f lib/Parser/Turtle.php         (Mateusz Żółtak     2018-07-19 16:25:56 +0200 1213)         $this->index -= $len;
545cb24f lib/Parser/Turtle.php         (Mateusz Żółtak     2018-07-19 16:25:56 +0200 1214)         foreach (preg_split('//u', $c, null, PREG_SPLIT_NO_EMPTY) as $h => $i) {
545cb24f lib/Parser/Turtle.php         (Mateusz Żółtak     2018-07-19 16:25:56 +0200 1215)             $this->data[$this->index + $h] = $i;
545cb24f lib/Parser/Turtle.php         (Mateusz Żółtak     2018-07-19 16:25:56 +0200 1216)         }
1b2f54a4 lib/EasyRdf/Parser/Turtle.php (Nicholas J Humfrey 2013-08-04 19:24:04 +0100 1217)     }
```

Reply:
Oh sorry, I am really confused by this diff

Reply:
Is this conversation solved?

Reply:
Is it solved? If so, please mark it.